srtp/srtp.c

```diff
@@ -4461,3 +4467,27 @@ srtp_err_status_t srtp_install_log_handler(srtp_log_handler_func_t func, void *
     }
     return srtp_err_status_ok;
 }
+
+srtp_err_status_t
+srtp_set_stream_roc(srtp_t session, uint32_t ssrc, uint32_t roc) {
+    srtp_stream_t stream;
+
+    stream = srtp_get_stream(session, htonl(ssrc));
+    if (stream == NULL)
+        return srtp_err_status_bad_param;
+
+    return srtp_rdbx_set_roc(&stream->rtp_rdbx, roc);
+}
+
+srtp_err_status_t
+srtp_get_stream_roc(srtp_t session, uint32_t ssrc, uint32_t *roc) {
+    srtp_stream_t stream;
+
+    stream = srtp_get_stream(session, htonl(ssrc));
+    if (stream == NULL)
+        return srtp_err_status_bad_param;
+
+    *roc = stream->rtp_rdbx.index >> 16;
+
```

Comment:
I fixed it with a new function, srtp_rdbx_get_roc. Let us wait for Paul's comments on how to proceed